Xamarin.Essentials/WebAuthenticator/AppleSignInAuthenticator.shared.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class AppleSignInAuthenticator
+    {
+        public static Task<WebAuthenticatorResult> AuthenticateAsync(AppleSignInOptions options = null)
+            => PlatformAuthenticateAsync(options ?? new AppleSignInOptions());
+
+        public class AppleSignInOptions
```

Comment:
I would maybe suggest having this a root class to match the other options. If we _really_ want to have a nested type, maybe just call it `Options` so the user types `AppleSignInAuthenticator.Options` instead of `AppleSignInAuthenticator.AppleSignInOptions`.